src/main/java/com/softserve/academy/spaced/repetition/service/AccountService.java

```diff
@@ -31,4 +31,17 @@ public void updateLearningRegime(LearningRegime learningRegime) throws NotAuthor
         account.setLearningRegime(learningRegime);
         accountRepository.save(account);
     }
+
+    public int getCardsNumber() throws NotAuthorisedUserException {
+        return userService.getAuthorizedUser().getAccount().getCardsNumber();
+    }
+
+    public void updateCardsNumber(Integer cardsNumber) throws NotAuthorisedUserException {
+        Account account = accountRepository.findOne(userService.getAuthorizedUser().getAccount().getId());
```

Comment:
After user updates the cards number, he has to re-login to use the new cards number?

Reply:
No, why? It works well.